packages/vscode/src/index.ts

```diff
@@ -37,12 +37,34 @@ export async function activate(ext: ExtensionContext) {
       ? toArray(root).map(r => path.resolve(projectPath, r))
       : [projectPath], config, status)
 
+  const skipMap = {
+    '<!-- @unocss-skip -->': ['<!-- @unocss-skip-start -->\n', '\n<!-- @unocss-skip-end -->'],
+    '// @unocss-skip': ['// @unocss-skip-start\n', '\n// @unocss-skip-end'],
```

Comment:
Can you also add `/* @unocss-skip-start */`? Thanks

Reply:
sure